c-dependencies/js-compute-runtime/xqd.h

```diff
@@ -224,163 +172,128 @@ int xqd_req_downstream_client_ip_addr_get(char *octets, size_t *nwritten);
+int xqd_req_method_get(fastly_request_handle_t req_handle, char *method, size_t method_max_len,
                        size_t *nwritten);
 
 WASM_IMPORT("fastly_http_req", "method_set")
-int xqd_req_method_set(RequestHandle req_handle, const char *method, size_t method_len);
+int xqd_req_method_set(fastly_request_handle_t req_handle, const char *method, size_t method_len);
 
 WASM_IMPORT("fastly_http_req", "uri_get")
-int xqd_req_uri_get(RequestHandle req_handle, char *uri, size_t uri_max_len, size_t *nwritten);
+int xqd_req_uri_get(fastly_request_handle_t req_handle, char *uri, size_t uri_max_len,
+                    size_t *nwritten);
 
 WASM_IMPORT("fastly_http_req", "uri_set")
-int xqd_req_uri_set(RequestHandle req_handle, const char *uri, size_t uri_len);
+int xqd_req_uri_set(fastly_request_handle_t req_handle, const char *uri, size_t uri_len);
 
 WASM_IMPORT("fastly_http_req", "version_get")
-int xqd_req_version_get(RequestHandle req_handle, uint32_t *version);
-
-WASM_IMPORT("fastly_http_req", "version_set")
-int xqd_req_version_set(RequestHandle req_handle, uint32_t version);
+int xqd_req_version_get(fastly_request_handle_t req_handle, uint32_t *version);
 
 WASM_IMPORT("fastly_http_req", "send")
-int xqd_req_send(RequestHandle req_handle, BodyHandle body_handle, const char *backend,
-                 size_t backend_len, ResponseHandle *resp_handle_out,
-                 BodyHandle *resp_body_handle_out);
+int xqd_req_send(fastly_request_handle_t req_handle, fastly_body_handle_t body_handle,
+                 const char *backend, size_t backend_len, fastly_response_handle_t *resp_handle_out,
+                 fastly_body_handle_t *resp_body_handle_out);
 
 WASM_IMPORT("fastly_http_req", "send_async")
-int xqd_req_send_async(RequestHandle req_handle, BodyHandle body_handle, const char *backend,
-                       size_t backend_len, PendingRequestHandle *pending_req_out);
+int xqd_req_send_async(fastly_request_handle_t req_handle, fastly_body_handle_t body_handle,
+                       const char *backend, size_t backend_len,
+                       fastly_pending_request_handle_t *pending_req_out);
 
 WASM_IMPORT("fastly_http_req", "send_async_streaming")
-int xqd_req_send_async_streaming(RequestHandle req_handle, BodyHandle body_handle,
-                                 const char *backend, size_t backend_len,
-                                 PendingRequestHandle *pending_req_out);
+int xqd_req_send_async_streaming(fastly_request_handle_t req_handle,
+                                 fastly_body_handle_t body_handle, const char *backend,
+                                 size_t backend_len,
+                                 fastly_pending_request_handle_t *pending_req_out);
 
-WASM_IMPORT("fastly_http_req", "pending_req_poll")
-int xqd_req_pending_req_poll(PendingRequestHandle req_handle, uint32_t *is_done_out,
-                             ResponseHandle *resp_handle_out, BodyHandle *resp_body_handle_out);
```

Comment:
I'm not 100% on this but I think this has been removed by mistake, is that correct?

Reply:
This is unused, so I removed the unused bindings, but I can appreciate we should leave them in!

Reply:
I've added back all removed xqd.h bindings except for the config-store ones which aliased the dictionary.